docs/linting/README.md

```diff
@@ -0,0 +1,5 @@
+# Linting
+
+Please be sure that you finished [The Linting Guide](/guide/linting.html) before you move on.
```

Comment:
"Please make sure to finish [The Linting Guide](/guide/linting.html) before continuing."